### PR TITLE
Remove github-actions bot from the contributors list

### PR DESCRIPTION
**Pull request by t-kikuc**
**What this PR does / why we need it**:

I modified the hack command to remove the following `github-actions` from the contributors list.

```html
<a href="https://github.com/apps/github-actions"><img src="https://avatars.githubusercontent.com/in/15368?v=4" title="github-actions[bot]" width="80" height="80"></a>
```

As I mentioned in https://github.com/pipe-cd/pipecd/pull/4987#discussion_r1648359978.

**Which issue(s) this PR fixes**: N/A

**Does this PR introduce a user-facing change?**: N/A

### CODE REVIEW

File: hack/gen-contributions.sh [hunk 1]
```diff
@@ -42,8 +42,9 @@ The Linux Foundation® (TLF) has registered trademarks and uses trademarks. For
 [![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fpipe-cd%2Fpipecd.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fpipe-cd%2Fpipecd?ref=badge_large)
 EOF
 
-# Remove fossabot from the contributors list.
+# Remove bots from the contributors list.
 sed -i '/fossabot/d' README.md.tmp
+sed -i '/github-actions[bot]/d' README.md.tmp
```
Comment (Warashi) [inline hack/gen-contributions.sh:47]: these symbols need to be escaped.

```suggestion
sed -i '/github-actions\[bot\]/d' README.md.tmp
```

Comment (t-kikuc) [reply hack/gen-contributions.sh]: @Warashi Thanks, I fixed it here-> [4b7604f](https://github.com/pipe-cd/pipecd/pull/4993/commits/4b7604f283cc0113c2e9ab2ef7e85d8395efbcbc)

**Review by Warashi — APPROVED** [review]
Thank you!
I checked the behavior on my Linux container with OrbStack, and it worked fine!

**Review by khanhtc1202 — APPROVED** [review]
Thanks :+1:

**Pull request merged**